packages/garbo-lib/src/wanderer/index.ts

```diff
@@ -247,8 +263,37 @@ export class WandererManager {
       : $location`Drunken Stupor`;
   }
 
-  getChoices(wanderer: WanderDetails): { [choice: number]: string | number } {
-    return this.unsupportedChoices.get(this.getTarget(wanderer)) ?? {};
+  /**
+   * Get choice map for the upcoming wander
+   * @param target Description of the wander or location in which to wander
+   * @param takeTurnForProfit Should the choices include any that would make a profit from your valueOfAdventure
+   * @returns Map of choice numbers to decisions
+   */
+  getChoices(
+    target: WanderDetails | Location,
+    takeTurnForProfit = this.options.takeTurnForProfit,
+  ): {
+    [choice: number]: string | number;
+  } {
+    const location =
+      target instanceof Location ? target : this.getTarget(target);
+    const choices = undelay(this.unsupportedChoices.get(location) ?? {});
+
+    if (takeTurnForProfit) {
+      const valueOfTurn =
+        (this.options.valueOfAdventure || 0) +
+        sum(getActiveEffects(), (e) => this.options.effectValue(e, 1));
+      if (location === $location`The Ice Hotel`) {
+        const valueOfCertificates = get("_iceHotelRoomsRaided")
+          ? 0
+          : this.options.itemValue($item`Wal-Mart gift certificate`) * 3;
+        if (valueOfCertificates > valueOfTurn) {
+          return { ...choices, 1116: 5 };
+        }
+      }
+    }
```

Comment:
I don't think I love the placement of this logic here. maybe delay `unsupportedChoices` with an `options` arg and put the logic inside there?